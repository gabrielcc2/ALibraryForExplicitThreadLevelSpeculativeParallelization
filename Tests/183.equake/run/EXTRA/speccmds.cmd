-u /home/gabriel/Escritorio/SPEC_CPU2000_CD_2/benchspec/CFP2000/183.equake/run/EXTRA 
-i inp.in >inp.out 2>inp.err ../EXTRA/equake
