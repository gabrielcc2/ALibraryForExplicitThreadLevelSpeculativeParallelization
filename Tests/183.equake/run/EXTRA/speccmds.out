running commands in speccmds.cmd 1 times
runs started at 1310404247, 210245000, Mon Jul 11 12:40:47 2011
run 1 started at 1310404247, 210464000, Mon Jul 11 12:40:47 2011
child started: 0, 1310404247, 210485000, 13488, '>inp.out 2>inp.err ../00000004/equake_base.x86_linux'
child finished: 0, 1310404337, 185034000, 89, 974549000, 13488, 0
run 1 finished at: 1310404337, 185046000, Mon Jul 11 12:42:17 2011
run 1 elapsed time: 89, 974582000, 89.974582000
runs finished at 1310404337, 185110000, Mon Jul 11 12:42:17 2011
runs elapsed time: 89, 974865000, 89.974865000
